packages/gatsby/src/utils/webpack/utils/create-normalized-module-key.ts

```diff
@@ -0,0 +1,25 @@
+import path from "path"
+
+/**
+ * Create a normalized module key that is referenced in both the partial hydration webpack loader and plugin.
+ * This solves for module imports that may be differently bundled for different environments (e.g. browser, node).
+ *
+ * If the module is a local module, the key is the relative path from the project root (e.g. `file://src/components/index.js`).
+ * If the module is a node module, the key is the relative path to the module name in `node_modules` (e.g. `file://node_modules/react`).
+ *
+ * @param resourcePath Absolute path to the resource
+ * @param rootContext Absolute path to project root
+ * @returns Normalized module key
+ */
+export function createNormalizedModuleKey(
```

Comment:
Talked to @wardpeet - we'll want to keep things _consistent_ so we should  use:

https://github.com/gatsbyjs/gatsby/blob/c073bbea40379adc62bd891da9f65ff2756b120b/packages/gatsby-core-utils/src/path.ts#L80-L94

It just converts everything to forward slash and then use that